c_api/impl/AuxIndexStructures_c.h

```diff
@@ -126,8 +126,24 @@ int faiss_RangeSearchPartialResult_set_lims(
 int faiss_RangeSearchPartialResult_new_result(
     FaissRangeSearchPartialResult* res, idx_t qno, FaissRangeQueryResult** qr);
 
+
+FAISS_DECLARE_CLASS(DistanceComputer)
+int faiss_DistanceComputer_new(FaissDistanceComputer** p_dc);
+int faiss_DistanceComputer_set_query(FaissDistanceComputer *dc, const float *x);
+int faiss_DistanceComputer_vector_to_query_dis(FaissDistanceComputer *dc, idx_t i, float *qd);
+/** 
+ * Compute distance between two stored vectors
+ *
+ * @param dc          opaque pointer to distance computer object
+ * @param key         output distance
```

Comment:
It's named `vd` below.

```suggestion
 * @param vd          output distance
```